.gitignore

```diff
@@ -1,8 +1,9 @@
+.idea
```

Comment:
Can you please leave a comment above explaining why this is here? I would love to have sections commenting so that out .gitignore is not hard to read or understand why something is there. Right now it's not a problem but can get harder once these grow.

Reply:
Sure